README.md

```diff
@@ -12,3 +12,34 @@
 - [How To Build](https://github.com/status-im/status-go/wiki/Build-Process-Explained)
```

Comment:
Leads to non-existing document in Git wiki while the existing one at https://github.com/status-im/status-go/wiki/Building is outdated.

Reply:
Oh, right. I didn't touch that part. Will link it to notion.so as it's most update to the date.